File: app/src/main/java/com/capstone/nik/mixology/Activities/ActivityDetails.java
package com.capstone.nik.mixology.Activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.Menu;

import com.capstone.nik.mixology.R;

public class ActivityDetails extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_details);

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        return true;
    }
}

File: app/src/main/java/com/capstone/nik/mixology/Activities/ActivityDetails.kt
package com.capstone.nik.mixology.Activities

import android.os.Bundle
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import com.capstone.nik.mixology.R
import com.capstone.nik.mixology.ui.getCocktailExtra
import com.capstone.nik.mixology.ui.details.DrinkDetailsRoute
import com.capstone.nik.mixology.ui.theme.MixologyTheme

class ActivityDetails : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val cocktail = intent.getCocktailExtra(getString(R.string.intent_details_intent_cocktail))
        if (cocktail == null) {
            finish()
            return
        }
        enableEdgeToEdge()
        setContent {
            MixologyTheme {
                DrinkDetailsRoute(
                    cocktail = cocktail,
                    showUpNavigation = true,
                    onBack = { finish() },
                )
            }
        }
    }
}
